### ISSUE TITLE
typo in docs: externaldnss

### ISSUE THREAD

**Issue by alkshmir** [F5Networks/k8s-bigip-ctlr#2810]
#### Description  
 In the [user guide](https://clouddocs.f5.com/containers/latest/userguide/kubernetes/) the `k8s_rbac.yaml` is like
```yaml
 - apiGroups: ["cis.f5.com"]
   resources: ["virtualservers","virtualservers/status", "tlsprofiles", "transportservers", "ingresslinks", "externaldnss"]
   verbs: ["get", "list", "watch", "update", "patch"]
```
but I suppose it should be like
```yaml
 - apiGroups: ["cis.f5.com"]
   resources: ["virtualservers","virtualservers/status", "tlsprofiles", "transportservers", "ingresslinks", "externaldnses"]
   verbs: ["get", "list", "watch", "update", "patch"]
```
#### Steps To Reproduce
1)
2)
3)

**Comment by trinaths** [comment]
No changes to schema. Its as intended.

**Comment by alkshmir** [comment]
Yes, the schema is not affected but the [documentation](https://clouddocs.f5.com/containers/latest/userguide/kubernetes/#installing-cis-manually) is likely incorrect. Where should I report it?

**Comment by trinaths** [comment]
Created [CONTCNTR-3841] for internal tracking.

**Comment by trinaths** [comment]
Fixed in 2.12.1
